contribute-to-tidb/write-document.md

```diff
@@ -224,6 +232,38 @@ You need to perform the following steps on each one of the cherry-picked PRs:
 
 After the steps above are completed, the administrator will merge the cherry-picked PRs. At this point, your contribution journey is completed! 🎉
 
+## How we implement bilingual documentation
```

Comment:
How can I apply for translation? From the content below all things start from `translation/doing` but as a contributor I'm interested in `translation/welcome` to apply for contribution?

Reply:
"translation/welcome" is seldom used now. I prefer letting the contributor pick one PR and assign himself/herself. See https://github.com/zz-jason/tidb-dev-guide/pull/40/commits/b38164f9f71b5809847d349ae579d5fa5c828b50.